share/plugins/colornotes.qml

```diff
@@ -116,15 +71,17 @@ MuseScore {
                         note.accidental.color = black;
                   }
 
-            for (var i = 0; i < note.dots.length; i++) {
-                  if (note.dots[i]) {
-                        if (note.dots[i].color == black)
-                              note.dots[i].color = colors[note.pitch % 12];
-                        else
-                              note.dots[i].color = black;
+            if (note.dots) {
+                  for (var i = 0; i < note.dots.length; i++) {
+                        if (note.dots[i]) {
+                              if (note.dots[i].color == black)
+                                    note.dots[i].color = colors[note.pitch % 12];
+                              else
+                                    note.dots[i].color = black;
+                              }
                         }
                   }
-         }
+      }
```

Comment:
this is where the excess indent from above looks bad and confusing

Reply:
Yes I noticed that myself, was going to fix